src/api/helpers/encrypt/index.ts

```diff
@@ -0,0 +1,30 @@
+import crypto from 'crypto';
+import { FINANCIAL_DETAILS } from '../../constants';
+import generateKey from './generate-key';
+import generateInitialisationVector from './generate-initialisation-vector';
+import { FinancialDetailsEncryption } from '../../types';
+
+const { ENCRYPTION_METHOD, ENCODING, STRING_ENCODING, OUTPUT_ENCODING } = FINANCIAL_DETAILS.ENCRYPTION.CIPHER;
+
+/**
+ * encrypt
+ * encrypts data
+ * returns encrypted value and initialisation vector
+ * @param dataToEncrypt
+ * @returns {FinancialDetailsEncryption} value and initialisation vector
```

Comment:
Not a blocker, can be done in next PR. This seems to be generic and could be used for non-financial things.

I think we should we change the return type to be `Encryption` or better `EncryptedData`, instead of `FinancialDetailsEncryption`.

Also don't think line 12 needs to be here since the `@returns` documentation says the exact same thing.

Lastly can we describe in line 11, _how_ it encrypts?